Fixed extra space in parent page for mobile devices

mohit-codecombat opened the pull request:
Fixed extra space in parent page for mobile device and add padding for course offering table.
1. Remove extra space from `Remove Learning` Section
![Live-coding-classes-from-CodeCombat-26](https://user-images.githubusercontent.com/75598717/102894586-59a31380-4489-11eb-8a3e-8ccec6aa0638.png)

2. Display `Pet following yellow dotted path` image which was not displaying before and causes extra space for "Course Offering" Section, Also rotate (-15 degree) this image to point `Course Offering` for mobile devices.
![Live-coding-classes-from-CodeCombat-27](https://user-images.githubusercontent.com/75598717/102895004-f49bed80-4489-11eb-829e-8f4c030b320d.png)

3. Added padding for `Course Offering` table.
![Live-coding-classes-from-CodeCombat-28](https://user-images.githubusercontent.com/75598717/102895036-05e4fa00-448a-11eb-93c8-a324bb14913d.png)

mohit-codecombat commented:
Adding Desktop version screenshot for above mobile devices screenshot.

![Live-coding-classes-from-CodeCombat-25](https://user-images.githubusercontent.com/75598717/102895799-5e68c700-448b-11eb-9d3c-cf4527914041.png)

![Live-coding-classes-from-CodeCombat-24](https://user-images.githubusercontent.com/75598717/102895809-6294e480-448b-11eb-836b-3df8ba365536.png)

AndrewJakubowicz commented on app/views/landing-pages/parents/PageParents.vue:
```diff
@@ -1333,6 +1336,23 @@ export default {
+    min-height: auto;
+  }
+  .pet-following-yellow-dotted {
+    margin-bottom: 0px; overflow-x: hidden;
```
Nitpick: Add a newline and keep each style on a separate line.

AndrewJakubowicz commented on app/views/landing-pages/parents/PageParents.vue:
```diff
@@ -1333,6 +1336,23 @@ export default {
+    transform: rotate(-30deg);
+    width: 75%;
+  }
+  .pricing-grid-container {
+    padding: 0 5px;
+  }
+  div.pricing-grid-container {
```
Do we need two selectors that appear to point at the same element?

AndrewJakubowicz commented:
Could you check with Shasha regarding rotating the asset? My opinion is that we'd rather translate it rather than rotate it as it looks out of place compared to the other graphics. We can also request a different asset, or switch to rotation on a media query.

Everything looks great, just interested in the clarifying the graphic.
Great work!

mohit-codecombat commented:
1. Removed image rotation.

![Live-coding-classes-from-CodeCombat-35](https://user-images.githubusercontent.com/75598717/103085969-461eb680-4609-11eb-8eb3-4228e63a48e1.png)

2. Fixed for image before "Student Outcomes" section.
![Live-coding-classes-from-CodeCombat-34](https://user-images.githubusercontent.com/75598717/103085974-47e87a00-4609-11eb-88e7-1031cdaadae7.png)

AndrewJakubowicz reviewed (approved):
Looks fantastic to me! I manually tested and also checked the TimeTap modal when you want to schedule an online class.

🚢 🚢 

I found another nitpick (image overlap with heading) but it doesn't have to be addressed in this pull request!
![image](https://user-images.githubusercontent.com/15080861/103314191-21468c80-49d7-11eb-9598-1cb24fb86a14.png)
